torch/__init__.py

```diff
@@ -1697,7 +1697,7 @@ def compile(model: Optional[Callable] = None, *,
 
         - Experimental or debug in-tree backends can be seen with `torch._dynamo.list_backends(None)`
 
-        - To register an out-of-tree custom backend: https://pytorch.org/docs/main/compile/custom-backends.html
+        - To register an out-of-tree custom backend: https://pytorch.org/docs/stable/torch.compiler_custom_backends.html
```

Comment:
This should be a cross-reference as so: https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html so that the code works with multiple versions of PyTorch.